code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -39,7 +39,7 @@
 				return FALSE
 
 			if(species?.species_flags & ROBOTIC_LIMBS)
-				to_chat(H, span_boldnotice("You cant help this one, [p_they()] have no lungs!"))
+				to_chat(H, span_boldnotice("You can't help this one, [p_they()] have no lungs!"))
```

Comment:
Also change the have to the macro too while you're at it. It often appears as `He have no lungs!`